src/client/elm/Explorer/View.elm

```diff
@@ -1888,7 +1888,7 @@ viewEventUpdateEnqueuedDetails ctx event =
                 ]
 
         FoundationAccountPayload foundationAccount ->
-            paragraph [] [ text "Update the Foundation account to be ", viewAddress ctx <| T.AddressAccount foundationAccount ]
+            paragraph [] [ text <| "Update the Foundation account to be acount with index " ++ String.fromInt foundationAccount ]
```

Comment:
```suggestion
            paragraph [] [ text <| "Update the Foundation account to be account with index " ++ String.fromInt foundationAccount ]
```

Reply:
Actually, I'd just say "Update the foundation account to index ..."

Reply:
I have changed the phrasing and fixed the above issue, it seems that the foundation account in the update payload is represented using index for the queued updates and as an account address in the transaction summary.